tests/terratest/tests/aks/aks_test.go

```diff
@@ -0,0 +1,92 @@
+package tests
+
+import (
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/josh-diamond/rancher-terratest/config"
```

Comment:
Is josh-diamond your directory in your go path?

Reply:
oof.. yes, yes it is

Reply:
truth comes out.. i drag and dropped this project from josh-diamond/rancher-terratest repo where i developed it.. sorry if complexities arose from this..

Reply:
i have a github.com > josh-diamond  rancher
i was developing this code in github.com > josh-diamond > rancher-terratest
the rancher/rancher repo, lives at github.com > rancher > rancher > tests > terratest

Reply:
whatever changes need to be made, im still not clear.  Just tell me whatever it is that needs to be done and i'll do my best to accomplish that 🙏

Reply:
No problem! So you should have cloned the rancher/rancher repo within a directory of this file structure. `path-to/go/src/github.com/rancher/rancher` Your GOPATH env var should set to `path-to/go`. So then when you are referencing these packages it would the correct path. In this case `"github.com/rancher/rancher/tests/terratest"` so on and so forth.